tests/_system/test_onedim_ckd_vs_mono.py

```diff
@@ -347,7 +355,9 @@ def test_1050(reflectance):
         mono_exp.run()
     mono_results = mono_exp.results["measure"]
 
-    wavelength_bin_width = (wavelengths[-1] - wavelengths[0]).m_as(mono_results.w.units)
+    wavelength_bin_width = (wavelengths.max() - wavelengths.min()).m_as(
+        mono_results.w.units
```

Comment:
Let's be explicit about the fact that this is metadata:
```suggestion
        mono_results.w.attrs["units"]
```